app/_src/guides/excluding-traffic-from-transparent-proxy-redirection.md

```diff
@@ -0,0 +1,229 @@
+
+- **ConfigMap**: In this guide, **ConfigMap** refers to the Kubernetes [**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/) resource. It’s used to store configuration data as key-value pairs that can be easily accessed by other resources in the cluster, such as Pods, Deployments, and StatefulSets.
+
+- **Service**: In this guide, **service** means the application running in a Universal mode environment where the transparent proxy will be installed. This term does not refer to Kubernetes [**Service**](https://kubernetes.io/docs/concepts/services-networking/service/) resources.
+
+## Prerequisites
+
+1. **Understand Transparent Proxy Basics**: You should already be familiar with what a transparent proxy is, how to install it, and how to adjust its settings. This guide won’t cover those basics. The following resources can help:
+
+   - For options to customize transparent proxy settings, check the resources listed in the [Key Information](#key-information) section.
+   - For an in-depth look at how the transparent proxy works in {{ Kuma }} and its mechanisms, see the [Transparent Proxying]({{ docs }}/networking/transparent-proxying/) documentation.
+   - To install the transparent proxy in Universal mode, refer to [Integrating Transparent Proxy into Your Service Environment]({{ path-integrating-tproxy-title }}).
+   - For upgrade steps, see [Upgrading Transparent Proxy]({{ docs }}/guides/upgrading-transparent-proxy).
+
+2. **Set Up {{ Kuma }}**: Ensure {{ Kuma }} is installed and running.
+
+   {% capture prerequisites-setup-kuma %}
+   {% tabs prerequisites-setup-kuma %}
+   {% tab prerequisites-setup-kuma Kubernetes %}
+   Follow the [Kubernetes Quickstart]({{ docs }}/quickstart/kubernetes-demo/) guide to set up a zone control plane and demo application.
+   {% endtab %}
+   {% tab prerequisites-setup-kuma Universal %}
+   Confirm that all necessary components are up and running. The following resources may be useful:
+
+  - To set up a single-zone control plane, follow the [Single Zone Control Plane Deployment]({{ docs }}/production/cp-deployment/single-zone/) guide.
+
+  - To generate a data plane proxy token (needed to authorize data plane proxies), refer to [Data Plane Proxy Token]({{ docs }}/production/secure-deployment/dp-auth/#data-plane-proxy-token) documentation.
+
+  - To prepare your service environment and start the data plane proxy, follow the [Integrating Transparent Proxy into Your Service Environment]({{ path-integrating-tproxy-title }}) guide up to [Step {% inc install-tproxy if_version="lte:2.4.x" init_value=5 %}: Install the Transparent Proxy]({{ path-step-install-tproxy }}).
+    {% endtab %}
+    {% endtabs %}
+    {% endcapture %}
+    {{ prerequisites-setup-kuma | indent }}
+
+## Exclude Traffic on Certain Ports
```

Comment:
I appreciate that we have key information, terminology and prerequisites but it might be a bit much for a basic use case right? I think this guide can be really entry level and lightweight with other guides going into more details.

Reply:
I actually think these might be helpful. Maybe if we would introduce a functionality so we can fold some sections, and even fold by default it would be better?